packages/suite/src/views/wallet/sign-verify/components/VerifyMessage/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect } from 'react';
+import styled, { keyframes } from 'styled-components';
+import { Input, Button, Textarea, variables } from '@trezor/components';
+import { Card, Translation } from '@suite-components';
+import { useForm } from 'react-hook-form';
+import { isAddressValid } from '@wallet-utils/validation';
+import { ChildProps as Props } from '../../Container';
+import { useActions } from '@suite-hooks';
+import * as signVerifyActions from '@wallet-actions/signVerifyActions';
+
+const AppearAnimation = keyframes`
+ 0% { opacity: 0; }
+ 10% { opacity: 0.1; }
+ 30% { opacity: 0.3; }
+ 70% {  opacity: 0.7;}
+ 80% {  opacity: 0.8;}
+ 90% {  opacity: 0.9;}
+ 100% { opacity: 1; }
+`;
+
+const Column = styled.div`
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: column;
+    animation-name: ${AppearAnimation};
+    animation-duration: 0.6s;
+    animation-iteration-count: 1;
+    @media screen and (max-width: ${variables.SCREEN_SIZE.MD}) {
+        flex: 1 1 100%;
+    }
+`;
+const Row = styled.div`
+    padding-bottom: 28px;
+    display: flex;
+`;
+const RowButtons = styled(Row)`
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+`;
+const StyledButton = styled(Button)`
+    width: 110px;
+    margin-left: 10px;
+
+    &:first-child {
+        margin-left: 0;
+    }
+`;
+type errorsMessage = 'TR_ADDRESS_IS_NOT_VALID' | 'TR_REQUIRED_FIELD';
+
+const VerifyMessage = ({ account, isLocked }: Props) => {
+    const { initVerify } = useActions({ initVerify: signVerifyActions.initVerify });
```

Comment:
why 'init' prefix? should be just `verify`, no?

Reply:
too much verify.... variables on page